tests/composite_fk/models/tenant.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Tenant(models.Model):
+    pass
+
+
+class User(models.Model):
+    tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("tenant_id", "id")
```

Comment:
I'm surprised this doesn't use a composite primary key. Was there a reason you chose not to?

Reply:
I figured we don't have to wait for composite primary keys to be merged to implement composite foreign keys.

Reply:
Ah - I didn't realise this was a completely stand-alone PR. At some point we'll need to check that the two PRs work together.

Reply:
Yes, depending on which one gets merged first the other will need to be adjusted.